app/controllers/hiring_staff/sign_in/dfe/sessions_controller.rb

```diff
@@ -11,47 +11,55 @@ def new
   end
 
   def create
-    Rails.logger.warn("Hiring staff signed in: #{oid}")
+    Rails.logger.warn("Hiring staff signed in: #{user_id}")
+    audit_successful_authentication
 
-    permissions = TeacherVacancyAuthorisation::Permissions.new
-    permissions.authorise(identifier, selected_school_urn)
-
-    log_succesful_authentication
-
-    if permissions.authorised?
-      update_session(permissions.school_urn, permissions)
+    authorisation = Authorisation.new(organisation_id: organisation_id, user_id: user_id).call
+    if authorisation.authorised?
+      update_session
       redirect_to school_path
     else
       not_authorised
     end
+  rescue Authorisation::ExternalServerError => error
+    Rollbar.log(:error, error)
```

Comment:
I wasn't able to figure out how to drive this line with tests but I am sure I want to make sure the team are alerted to failures through Rollbar. Any ideas?

Reply:
Could you do something like `allow_any_instance_of(Authorisation).to recieve(:call) { raise Authorisation::ExternalServerError }`?